### PR TITLE
chore(git): update .gitignore

### PR DESCRIPTION
**Pull request by LiangliangSui**
```
On branch main
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	MODULE.bazel
	MODULE.bazel.lock

nothing added to commit but untracked files present (use "git add" to track)
```

Update `.gitignore` to ignore two intermediate files.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -34,4 +33,6 @@ bazel-fury/**
 java/**/generated
 javascript/**/dist/
 javascript/**/node_modules/
-javascript/**/build
\ No newline at end of file
+javascript/**/build
+MODULE.bazel
```
Comment (chaokunyang) [inline .gitignore:37]: Is this generated in windows?

Comment (LiangliangSui) [reply .gitignore:37]: No, ubuntu.

Comment (LiangliangSui) [reply .gitignore:37]: ubuntu 22.04

**Review by chaokunyang — APPROVED** [review]
LGTM

**Pull request merged**